packages/block-library/src/spacer/edit.js

```diff
@@ -58,7 +58,7 @@ const SpacerEdit = ( { attributes, isSelected, setAttributes, toggleSelection, i
 									height: parseInt( event.target.value, 10 ),
 								} );
 							} }
-							value={ height }
+							value={ isNaN( height ) ? '' : height }
```

Comment:
Instead of passing an empty string, you can pass `20` if height is NaN, since the min value of the input field is set to 20. ( check line 62 )